client/components/utils/buildProductQuery.js

```diff
@@ -0,0 +1,13 @@
+export default ({ productPagination, productFilters }) => {
+  let query = [];
+  for (const paramKey in productPagination) {
+    if (paramKey !== 'maxPage') {
+      query.push(`${paramKey}=${productPagination[paramKey]}`); //ex. page=1
+    }
+  }
+  for (const paramKey in productFilters) {
+    query.push(`${paramKey}=${productFilters[paramKey]}`);
+  }
+  query = `?${query.join('&')}`;
+  return query;
+};
```

Comment:
This is a utility function used to build a `query` string from the `productPagination` and `productFilter` objects in our store.